intro-to-object-oriented-programming/what-is-oop.md

```diff
@@ -128,19 +129,29 @@ We might say...
 
 ### State and Behavior
 
-When we start defining our own classes, we will specify their **state** and their generic **behavior.**
+When we start defining our own classes, we will specify two main things: their **state** and their **behavior**.
+
+In the context of Object-Oriented Programming, **state** is used to refer to all the variables packaged within a class and **behavior** is used to refer to all the functions packaged within a class.
+
+### Attributes and Methods
+
+There are two more words we will often see in the context of OOP: **attribute** and **method**.
+
+For whatever reason, engineers long-ago decided we needed new words to describe variables and functions that make up a class's state and behavior. 
+
+An **attribute** is just a specific type of **variable**: one defined within a class. Similarly a **method** is just a specific type of **function**: one defined within a class.
```

Comment:
Nitpick:

I'm gonna push to find another word besides "just"! I'd really like "just" to be used for the feeling of "we think students will do more so do _just_ enough." (I wonder if in my previous comment, I contradicted myself and said this usecase "just" was cool?)

I won't "request changes" and I think you're ready to merge, but I'm interested in seeing if you happen to like an alternative:

```suggestion
An **attribute** is really a specific type of **variable**: one defined within a class. Similarly a **method** is really a specific type of **function**: one defined within a class.
```

```suggestion
**Attributes** are a specific kind of **variable**: ones defined within a class. Similarly **methods** are a specific kind of **function**: ones defined within a class.
```

First one: I realize it's replacing "just" with another word that means the same. /shrug. I feel more strongly against "just" than "really" lol.

Second one: I wonder if the "just" can get across by making the sentence a lil more direct about the definition. Also, I'm suggesting the word "kind" bc "kind" usually implies that it's still classified also as a variable/function, where "type" softly implies exclusivity (it's either a function or a method). ("kind vs. type" is not something I feel strongly about)

Reply:
Love it! There's some funkiness in the second suggestion with singular/plural but I'll adjust it a little and push that up!